SHSearchBar/Classes/SHSearchBar.swift

```diff
@@ -267,7 +270,7 @@ public class SHSearchBar: UIView, SHSearchBarDelegate {
     }
 
     @objc func pressedCancelButton(_ sender: AnyObject) {
-
+        cancelButtonDidClick?()
```

Comment:
I implemented the delegate `searchBarShouldCancel()`. Why don't you use this since it is working and tested.